openfish-webapp/README.md

```diff
@@ -0,0 +1,20 @@
+# Openfish Webapp
+Openfish Webapp is a web interface for interacting wit the Openfish API.
+
+## Development guide
+#### Getting started
+
+Run the commands:
+```
+cd openfish-webapp
+pnpm i
```

Comment:
It would be useful to put the dependencies in here so that people can build and run without too much hassle. Ideally I think we should move towards using docker and write a Dockerfile for this repo.

Reply:
That is a good idea, especially given that some JS features will only work with a semi-recent version of node

Reply:
I have added docker support and listed dependencies here.